lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -71,14 +70,17 @@ function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL) {
     'file' => isset($values['file']) ? $values['file'] : NULL,
     'caption' => isset($values['caption']) ? $values['caption'] : NULL,
     'status' => isset($values['status']) ? $values['status'] : 'pending',
-    'crop_x' => $values['crop_x'],
-    'crop_y' => $values['crop_y'],
-    'crop_width' => $values['crop_width'],
-    'crop_height' => $values['crop_height'],
-    'crop_rotate' => $values['crop_rotate'],
     'source' => isset($values['source']) ? $values['source'] : NULL,
   ];
 
+  if ($values['crop_x']) {
```

Comment:
ah yes, great call! Since the two are structured a bit differently (ie `$data['crop_x']` vs `$values['crop_x']` and are set to an array vs object `$values['crop_x']` vs `$task->crop_x`) would it be cleanest to write a helper function with conditionals checking if it's in the `$data` format, set to array, else set to the `$task` object?